src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.cs

```diff
@@ -822,7 +946,7 @@ private static ISymbol GetEventAccessor(IEventSymbol iEvent, IOperation operatio
                         {
                             // Call site has no attributes for this platform, check if MsBuild list has it,
                             // then if call site has deny list, it should support its later support
-                            if (msBuildPlatforms.Contains(platformName) &&
+                            if (msBuildPlatforms.IndexOf(platformName, 0, StringComparer.OrdinalIgnoreCase) != -1 &&
```

Comment:
These are platform names from MSBuild plus custom platforms added using `<SupportedPlatform Include="browser"/>`, technically it could match substring, but i don't recall any platform could be sub string of others

Reply:
It would be safer to guard against the possibility of false-positive substring matches; it's possible that years from now, such a scenario will occur, and this code would be hard to track down.